cli/src/cluster_query.rs

```diff
@@ -768,6 +770,18 @@ pub fn process_catchup(
             )
         };
 
+        let slot_distance_avg = node_slot as i64 - first_node_slot as i64;
+        let slots_per_second_avg = slot_distance_avg as f64 / f64::from(interval_count * sleep_interval);
```

Comment:
(continuing from https://github.com/solana-labs/solana/pull/15608/files#r586499806) then this becomes:

```rust
let average_slots_per_second = total_slot_distance as f64 / f64::from(total_sleep_interval);
```

Reply:
note that I'm preferring to `average_` than `_avg`, not big issue, though. :)